app.js

```diff
@@ -22,4 +22,25 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi => {
+      console.log('Beers from the database: ', beersFromApi);
+      res.render('beers', { beersFromApi })
+    })
+    .catch(error => console.log(error));
+
+})
+
+app.get('/random-beer', (req, res) => {
+  punkAPI
+    .getRandom()
+    .then(randomBeer => {
+      console.log('Random beer from the database: ', randomBeer);
+      res.render('randombeer', { randomBeer })
```

Comment:
You correctly handled this as an array in the hbs using #each. Since this will always be an array of 1 element you could alternatively pass ` randomBeer[0]` to res.render and then you only have a single element to handle in the hbs